packages/database/tests.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
It is unlikely we will using `ipynb`'s as a standard part of our workflow, so let's not commit this to the repo.

If you want to use this locally for yourself to confirm routes are working and such, that is fine  In that case, let's add `tests.ipynb` to the `gitignore`.

Reply:
done.